tools/cli/go-whisk/whisk/activation.go

```diff
@@ -70,9 +70,13 @@ type Log struct {
     Time   string `json:"time,omitempty"`
 }
 
+const USE_DEFAULT_NAMESPACE = true
```

Comment:
@mdeuser, this is temporary until we can use non-default namespaces with the activation commands.